lib/BackgroundJob/Tasks/ImageProcessingTask.php

```diff
@@ -389,7 +402,7 @@ private function calculateMaxImageArea(): int {
 		// Based on amount on memory PHP have, we will determine maximum amount of image size that we need to scale to.
 		// This reasoning and calculations are all based on analysis given here:
 		// https://github.com/matiasdelellis/facerecognition/wiki/Performance-analysis-of-DLib%E2%80%99s-CNN-face-detection
-		$maxImageArea = intval(($allowedMemory) / SettingsService::MEMORY_AREA_RELATIONSHIP); // TODO: Maybe another helper.
+		$maxImageArea = intval(($allowedMemory) / $model->getMemoryAreaRelation()); // TODO: Maybe another helper.
```

Comment:
I would not expose `getMemoryAreaRelation` as something model have. I would rather expose something like `model::GetRecommendedImageArea` and keep memory are relation and such things deep in model logic. Some models will have memory are relations, some will not have it. Some will have bandwidth relation (remote service model) or something else, so maybe it makes sense to expose just what a model is "recommending" as max area

Reply:
I totally agree, but my doubts is the front. I can't generalize it so much. Maybe should remove it and ask questions in the setup command.
Ideally, send the path of the real image, and have the model resize it if necessary.. But I would leave it for another PR.